tests/tests/test_pytest.py

```diff
@@ -0,0 +1,12 @@
+from star_tides.tests.fixtures.basic_user import basic_user
+from star_tides.services.mongo.models.user_model import User
+
+
+def test_basic_test(client, basic_user):
```

Comment:
Question: can we put tests in the same files as their related sources? Or is that not how Flask apps are done?

Reply:
I'd put them in a file in the same dir, but I don't think we should put them in the same file.

Reply:
Whoops, yeah I meant directory 🤦‍♂️ 

Something like:

.../services/users.py
.../services/users_test.py

Just so it's clearer where tests files live and what our coverage is. It may also help with scoping/visibility of things (what needs to be private vs public), but this could just be a bazel-ism.

Reply:
Naming convention is `test_<name of file>`.  Routes don't need to be tested because they're lean as a bean and literally just call controllers. Controllers will be the "end to end" tests. Actions and Interfaces can have tests written as well.